src/pages/App.js

```diff
@@ -0,0 +1,17 @@
+import Auth from "../Components/auth";
```

Comment:
Hi Can you change the file name from App.js to Login.js, It will cause confusion.